src/Cuboid.cpp

```diff
@@ -24,10 +24,10 @@ static bool DoIntervalsIntersect(int a_Min1, int a_Max1, int a_Min2, int a_Max2)
 ////////////////////////////////////////////////////////////////////////////////
 // cCuboid:
 
-cCuboid & cCuboid::operator =(cCuboid a_Other)
+cCuboid & cCuboid::operator =(const cCuboid & a_Other)
```

Comment:
There needs to be an explicit copy assignment operator because there's a user-declared copy constructor (and clang will complain), but I didn't see why there had to be a user-declared copy constructor so I deleted that.